### PR TITLE
Mongo feature s390x

### PR DESCRIPTION
**Pull request by Rajeshri-Shil**
Kindly review the changes for mongodb for s390x.

### CODE REVIEW

File: python/src/mas/cli/install/params.py [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "dro_namespace",
     # MongoDb
     "mongodb_namespace",
+    "mongodb_action"
```
Comment (whitfiea) [inline python/src/mas/cli/install/params.py]: missing a comma

File: python/src/mas/cli/install/argParser.py [hunk 2]
```diff
@@ -292,6 +292,14 @@ def isValidFile(parser, arg) -> str:
     required=False,
     help=""
 )
+mongoArgGroup.add_argument(
+    "--mongodb-action",
+    dest="mongodb_action",
+    required=False,
+    help="BYO MongoDB datasource ",
+    action="store_const",
+    const="byo"
```
Comment (whitfiea) [inline python/src/mas/cli/install/argParser.py]: Why does it have byo?

File: python/src/mas/cli/install/argBuilder.py [hunk 3]
```diff
@@ -132,8 +132,12 @@ def buildCommand(self) -> str:
 
         # MongoDb Community Operator
         # -----------------------------------------------------------------------------
-        if self.getParam('mongodb_namespace') != "":
-            command += f"  --mongodb-namespace \"{self.getParam('mongodb_namespace')}\"{newline}"
+        if self.getParam('mongodb_action') == "install" and self.getParam('architecture') != "s390x":
```
Comment (whitfiea) [inline python/src/mas/cli/install/argBuilder.py:135]: what about uninstall?

File: python/src/mas/cli/install/argBuilder.py [hunk 4]
```diff
@@ -132,8 +132,12 @@ def buildCommand(self) -> str:
...
+            if self.getParam('mongodb_namespace') != "":
+                command += f"  --mongodb-namespace \"{self.getParam('mongodb_namespace')}\"{newline}"
+                command += f"  --mongodb-action{newline}"
+        elif self.getParam('mongodb_action') == "byo":
```
Comment (whitfiea) [inline python/src/mas/cli/install/argBuilder.py]: byo isn't an action i recognise https://github.com/ibm-mas/ansible-devops/blob/35017863e41ff139d567671a507800c49b4f1bc5/ibm/mas_devops/roles/mongodb/README.md?plain=1#L48

**Review by whitfiea — CHANGES_REQUESTED** [review]
Can you add a description to the PR and update the title as to what this change is and what it is for, and what testing has happened?

**Comment by Rajeshri-Shil** [comment]
closin PR as the above changes does not require now.

**Pull request closed without merging**